demisto_sdk/commands/openapi_codegen/openapi_codegen.py

```diff
@@ -0,0 +1,837 @@
+            all_security_types = [s.get('type') for s in self.security_definitions.values()]
+            security_types = [t for t in all_security_types if t in [API_KEY_AUTH_TYPE, CREDENTIALS_AUTH_TYPE]]
+        if not security_types:
+            security_types = [API_KEY_AUTH_TYPE]
+
+        configuration = {
+            'name': self.name or 'GeneratedIntegration',
+            'description': self.description or 'This integration was auto generated by the Cortex XSOAR SDK.',
+            'category': 'Utilities',
+            'url': self.host or DEFAULT_HOST,
+            'auth': security_types,
+            'context_path': self.context_path,
+            'commands': []
+        }
+        for function in self.functions:
+            command = {
+                'name': function['name'].replace('_', '-'),
+                'path': function['path'],
+                'method': function['method'],
+                'description': function['description'],
+                'arguments': [],
+                'outputs': [],
+                'context_path': function.get('context_path', ''),
+                'root_object': function.get('root_object', '')
+            }
+            headers = []
+            if function['consumes']:
+                headers.append({'Content-Type': JSON_TYPE_HEADER
+                                if JSON_TYPE_HEADER in function['consumes'] else function['consumes'][0]})
+            if function['produces']:
+                headers.append({'Accept': JSON_TYPE_HEADER
+                                if JSON_TYPE_HEADER in function['produces'] else function['produces'][0]})
+
+            command['headers'] = headers
+            for arg in function['arguments']:
+                command['arguments'].append({
+                    'name': str(arg.get('name', '')),
+                    'description': self.clean_description(arg.get('description', '')),
+                    'required': arg.get('required'),
+                    'default': arg.get('default', ''),
+                    'in': arg.get('in', ''),
+                    'ref': arg.get('ref', ''),
+                    'type': arg.get('type', 'string'),
+                    'options': arg.get('enums'),
+                    'properties': arg.get('properties', {})
+                })
+
+            for output in function['outputs']:
+                command['outputs'].append({
+                    'name': output['name'],
+                    'description': output['description'],
+                    'type': output['type']
+                })
+
+                if output['name'] in self.unique_keys:
+                    command['unique_key'] = output['name']
+
+            if 'unique_key' not in command:
+                command['unique_key'] = ''
+
+            configuration['commands'].append(command)
+
+        configuration['code_type'] = 'python'
+        configuration['code_subtype'] = 'python3'
+        configuration['docker_image'] = 'demisto/python3:3.8.3.9324'

```

Comment:
lets use `get_docker_image_latest_tag()` here instead

Reply:
can't, that function is not generic enough

Reply:
how much effort is it to make it work? 
if we wont do it, this means the validation will always fail (not big deal but if we can avoid it, then it would be better)